### PR TITLE
gh-1980 Fixed crash when old request is retained

### PR DESCRIPTION
**Pull request by DmitryBespalov**
Handles #1980 

Changes proposed in this pull request:
- Fixing crash

### CODE REVIEW

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/App/MainTabBarViewController.swift [hunk 1]
```diff
@@ -503,6 +503,11 @@ extension MainTabBarViewController: WebConnectionRequestObserver {
         guard !requestQueue.isEmpty && !presentingRequest else { return }
         let request = requestQueue.removeLast()
 
+        guard let connection = WebConnectionController.shared.connection(for: request), connection.status == .opened else {
```
Comment (MouazAlzahabi) [inline Multisig/UI/App/MainTabBarViewController.swift:508]: @DmitryBespalov Would be nice to add small comment here why is this needed

**Review by MouazAlzahabi — APPROVED** [review]

File: Multisig/UI/App/MainTabBarViewController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/UI/App/MainTabBarViewController.swift:508]: ok
